packages/react-magma-dom/src/components/Modal/Modal.tsx

```diff
@@ -372,89 +342,87 @@ export const Modal = React.forwardRef<HTMLDivElement, ModalProps>(
       <CloseIcon color={theme.colors.neutral03} size={theme.iconSizes.small} />
     );
 
-    return isModalOpen
-      ? ReactDOM.createPortal(
-          <>
-            <Global
-              styles={css`
-                html {
-                  overflow: hidden;
-                }
-              `}
-            />
-
-            <ModalContainer
-              aria-labelledby={headingId}
-              aria-modal={true}
-              data-testid={testId}
-              id={id}
-              onKeyDown={isEscKeyDownDisabled ? null : handleKeyDown}
-              onClick={isBackgroundClickDisabled ? null : handleModalClick}
-              onMouseDown={
-                isBackgroundClickDisabled ? null : handleModalOnMouseDown
-              }
-              ref={focusTrapElement}
-              role="dialog"
-              theme={theme}
-            >
-              <ModalContent
-                {...other}
-                data-testid="modal-content"
-                id={contentId}
-                isExiting={isExiting}
-                isInverse={false}
-                ref={ref}
-                theme={theme}
-              >
+    return (
+      <>
+        <Global
+          styles={css`
+            html {
+              overflow: hidden;
```

Comment:
It looks like this is getting applied even when the modal or drawer isn't open.  As a result, I can't scroll on the modal and drawer docs pages.